Parcare/Services/BankAccountService.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Parking.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Parking.Services
+{
+    public class BankAccountService
+    {
+        public List<BankAccountModel> accounts = new List<BankAccountModel>();
+
+        public void Add(string name, List<string> carNumber, float balance)
+        {
+            accounts.Add(new BankAccountModel { Name = name, CarNumber = carNumber, Balance = balance });
+        }
+
+        public void Remove(string carNumber)
+        {
+            var accountsWithCarNumber = accounts.FirstOrDefault(account => account.CarNumber.Contains(carNumber));
+            if (accountsWithCarNumber != null)
+            {
+                accounts.Remove(accountsWithCarNumber);
```

Comment:
you could return after line 24 and not have the else anymore (again save the nesting elements and it will be easier to read)